gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
This seems to be a generated file, generated files, should not be uploaded to repository.